stable/coredns/README.md

```diff
@@ -47,7 +48,7 @@ The command removes all the Kubernetes components associated with the chart and
 | Parameter                               | Description                                                                           | Default                                                     |
 |:----------------------------------------|:--------------------------------------------------------------------------------------|:------------------------------------------------------------|
 | `image.repository`                      | The image repository to pull from                                                     | coredns/coredns                                             |
-| `image.tag`                             | The image tag to pull from                                                            | `v1.7.1`                                                    |
+| `image.tag`                             | The image tag to pull from                                                            | `1.8.0`                                                    |
```

Comment:
This is not true. Can you remove it ?

Reply:
@haad Both the charts which are tagged have 1.8.0 as default in the `values.yaml ` Not sure where the v1.7.1is based out on this repository